portfolio/src/main/webapp/portfolio.html

```diff
@@ -19,6 +19,7 @@ <h1 class="header">Portfolio</h1>
       <ul>
         <li><a href="index.html">About</a></li>
         <li><a href="#" id="here">Portfolio</a></li>
+        <li><a href="comments.html" id="here">Comments</a></li>
```

Comment:
what's the id here for?

Reply:
Nothing right now, it's a placeholder for further styling on the navbar that I plan to implement in the future.